airflow/contrib/hooks/gcp_compute_hook.py

```diff
@@ -84,20 +85,24 @@ def start_instance(self, zone, resource_id, project_id=None):
         :type project_id: str
         :return: None
         """
-        response = self.get_conn().instances().start(
-            project=project_id,
-            zone=zone,
-            instance=resource_id
-        ).execute(num_retries=NUM_RETRIES)
         try:
+            response = self.get_conn().instances().start(
+                project=project_id,
+                zone=zone,
+                instance=resource_id
+            ).execute(num_retries=NUM_RETRIES)
             operation_name = response["name"]
         except KeyError:
             raise AirflowException(
                 "Wrong response '{}' returned - it should contain "
                 "'name' field".format(response))
-        self._wait_for_operation_to_complete(project_id=project_id,
-                                             operation_name=operation_name,
-                                             zone=zone)
+        except errors.HttpError as ex:
```

Comment:
I think I have changed the way how googleapiclient  HttpError is imported in last commit

```
from googleapiclient.errors import HttpError
```

 Please take a look. 

It was causing strange errors when generating documentation. The way I did imports is better for sphinx it seems.